build/parseChangelog.c

```diff
@@ -178,13 +179,17 @@ static int dateToTimet(const char * datestr, time_t * secs, int * date_words)
 
     if (*secs == -1) goto exit;
 
+    if (tzname[1][0] == 0)
+	rpmlog(RPMLOG_WARNING, _("bogus TZ database name in %%changelog: %s\n"), datestr);
```

Comment:
Why the talk about *database* here? It's the timezone name that is wrong, not a name of some database as the message indicates.

Reply:
Corrected in the new version.